src/parser/permit-generation-module.ts

```diff
@@ -53,6 +54,11 @@ export class PermitGenerationModule implements Module {
       console.warn("[PermitGenerationModule] Invalid env detected, skipping.");
       return Promise.resolve(result);
     }
+    const isPrivateKeyAllowed = await this._isPrivateKeyAllowed(payload.evmPrivateEncrypted, program.eventPayload.organization?.id || 0, program.eventPayload.repository.id);
```

Comment:
so this plugin won't work for repos without organization? I know this is an edge case but we could easily make it work if we use `eventPayload.repository.owner.id` which is org or user

Reply:
Fixed https://github.com/ubiquibot/conversation-rewards/pull/111/commits/0d343459bdd42f23c22077c8fc9d2ddce46fa2ca